src/components/ClaimENSPage/ClaimENSPage.container.ts

```diff
@@ -3,15 +3,19 @@ import { push, goBack } from 'connected-react-router'
 import { getAddress } from 'decentraland-dapps/dist/modules/wallet/selectors'
 import { RootState } from 'modules/common/types'
 import { openModal } from 'modules/modal/actions'
+import { claimNameRequest } from 'modules/ens/actions'
+import { getENSByWallet } from 'modules/ens/selectors'
 import { MapStateProps, MapDispatchProps, MapDispatch } from './ClaimENSPage.types'
 import ClaimENSPage from './ClaimENSPage'
 
 const mapState = (state: RootState): MapStateProps => ({
+  ensList: getENSByWallet(state), // this need to be replaced by all the ENS
```

Comment:
Careful here! this is not what I mean. Downloading all the ENSs is not really feasible! the idea is to check it using thegraph on the saga, the issue has it well explained I think